### ISSUE TITLE
Zoomable week view

### ISSUE THREAD

**Issue by freswa** [marcoancona/MineTime#68]
Thanks for this great calendar tool :)
I'd like to zoom in and out of the week view (only vertical) to have a better overview/to see more details of the days. Could be done via Ctrl+Scroll or Ctrl+(+/-).

**Comment by marcoancona** [comment]
Hi, thanks for the suggestion. We are considering this. In the meanwhile, are you aware of the option "slot duration" in the app settings? With this you can select 3 different zoom levels.

**Comment by freswa** [comment]
Yes, I am aware of that setting. But I need different zoom levels while scrolling through my calendar. E.g. for an overview over the day I want to scroll out to see all events for that day but when looking at a conference schedule I need to zoom in since every slot is 0.5-1h long. 
Opening the settings every time is a bit cumbersome and not very intuitive opposed to the zoom feature on iOS/macOS.
Thanks for considering this.
